src/Extension/ProductInventoryManager.php

```diff
@@ -104,7 +112,21 @@ public function getNumberPurchased()
     public function getOrders()
```

Comment:
Seems like an opportunity to make this more efficient

Reply:
I think there are some efficiencies that could be gained, but that can be a good next release ticket.

Reply:
👍